test/e2e/__init__.py

```diff
@@ -38,6 +39,17 @@ def sagemaker_client():
     return boto3.client("sagemaker")
 
 
+def cross_region_sagemaker_client():
```

Comment:
can instead modify the existing method?

```
def sagemaker_client(region: str = None):
    region = region or get_region()
    return boto3.client("sagemaker", region_name=region)
```


```
def get_sagemaker_model(config_name: str, sm_client = None):
  sm_client = sm_client or sagemaker_client()
  ...
```

so now you can create the client in the test file and pass it to these methods